microceph/ceph/services_placement_generic.go

```diff
@@ -63,17 +66,17 @@ func genericHospitalityCheck(service string) error {
 	return nil
 }
 
-func genericServiceInit(s interfaces.StateInterface, name string) error {
+func genericServiceInit(s interfaces.StateInterface, name string, isClientService bool) error {
```

Comment:
I wonder if instead of adding a flag to the genericServiceInit the rbd-mirror service could subclass GenericServicePlacement, like the RGW service placement does? This would feel cleaner